source/developers-guide/shopware-composer/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Using Composer to install Shopware
+github_link: developers-guide/shopware-composer/index.md
+indexed: true
+group: Developer Guides
+subgroup: General Resources
+menu_title: Using composer with Shopware
+menu_order: 45
+---
+
+<div class="toc-list"></div>
+
+## Introduction
+
+Starting with v5.4 Shopware supports installing a shop as a composer dependency out of the box. This helps you to
+professionalize development and deployments of Shopware shops by providing a reliable versioning of Shopware itself and
+all the plugins required by your project.
+
+## What is composer
+
+Composer defines itself as "[...] a tool for dependency management in PHP. It allows you to declare the libraries
+your project depends on and it will manage (install/update) them for you" ([Composer website](https://getcomposer.org/)).
+
+Composer is the de-facto-standard for dependency management in the PHP community. You can think of it as the
+apt/yum/npm/brew of the PHP world.
+
+## How to start
+
+To create a new Shopware project with Shopware as a dependency, all you need to do is run
```

Comment:
for that composer needs to be installed of course. I would mention that :smile:

Reply:
Good point